modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/introspection/RuntimeSnapshot.java

```diff
@@ -94,26 +95,29 @@
 
     protected final List<Class<?>> spi = new ArrayList<>();
 
+    protected boolean pluginSnapshotsInitialized = false;
+
+    protected final Map<String, PluginSnapshot<?>> pluginSnapshots = new HashMap<>();
+
     public static RuntimeSnapshot build() {
         return new RuntimeSnapshot();
     }
 
     @JsonCreator
     private RuntimeSnapshot(@JsonProperty("serverInfo") ServerInfo serverInfo,
-            @JsonProperty("creationDate") Date created,
-            @JsonProperty("seamComponents") List<SeamComponentInfo> seamComponents,
-            @JsonProperty("operations") List<OperationInfo> operations) {
+            @JsonProperty("creationDate") Date created, @JsonProperty("operations") List<OperationInfo> operations,
+            @JsonProperty("pluginSnapshots") Map<String, PluginSnapshot<?>> pluginSnapshots) {
         this.serverInfo = serverInfo;
         this.created = created;
         index();
-        this.seamComponents.addAll(seamComponents);
         this.operations.addAll(operations);
+        opsInitialized = true;
+        this.pluginSnapshots.putAll(pluginSnapshots);
     }
```

Comment:
actually the json export is completely buggy and was never exposed publicly, only in tests... i'm working on exposing it, so i will fix it (without compatibility with previous format, then) -- more commits coming about this in the same PR (as i realized the problem when adding more unit tests)

Reply:
note i modified to the parser so that it's resilient to "unknown" fields at read time, and i will attach an old json serialization for history in the jira issue